packages/manager/src/components/RegionSelect/RegionSelect.utils.ts

```diff
@@ -205,3 +205,13 @@ export const useIsGeckoEnabled = (createType: LinodeCreateType) => {
       !supportedEdgeTypes.includes(createType as SupportedEdgeTypes)
   );
 };
+
+export const filterOutCurrentRegionAndCoreRegions = (
```

Comment:
Maybe we can prefix the function with `get`. Open to other ideas

```suggestion
export const getRegionsWithoutCurrentRegionAndCoreSites = (
```